### PR TITLE
Passes tx hash as route param to Verified component [Fixes #310]

### LINKED ISSUE #310: BrightID verification success page links to fake tx hash
The tx we link out to Etherscan is incorrect:

clr fund - clr fund 2021-08-27 at 3 38 40 PM

You can see here we never implemented this functionality:
https://github.com/ethereum/clrfund/blob/develop/vue-app/src/views/Verified.vue#L51

This should mimic the tx success page, getting the hash from the route param:
https://github.com/ethereum/clrfund/blob/develop/vue-app/src/router/index.ts#L186
Then grabbing that for the component:
https://github.com/ethereum/clrfund/blob/develop/vue-app/src/views/TransactionSuccess.vue#L50

### PR DESCRIPTION
**Pull request by wackerow**
### Description
- Upon completion of BrightID verification, the tx hash is passed when navigating to the Verified component (`/verify/success/:hash`)
- Hash is then used to properly link to block explorer

### Related issues
[Fixes #310]

### CODE REVIEW

File: vue-app/src/views/Verified.vue [hunk 1]
```diff
@@ -11,7 +11,7 @@
           <span class="emoji">🎉</span>
           <div class="flex-title">
             <h1>Ready to contribute!</h1>
-            <links v-if="txHash" :to="blockExplorerUrl">
+            <links v-if="$route.params.hash" :to="blockExplorerUrl">
```
Comment (corwintines) [inline vue-app/src/views/Verified.vue]: Would it make sense to switch this to TransactionReceipt for consistency around other transactions (reallocation, registration, contribution). Other than that this is working great!

Comment (samajammin) [reply vue-app/src/views/Verified.vue]: Agreed, let's use that component here.

**Review by samajammin — CHANGES_REQUESTED** [review]
See comment 👍

**Review by corwintines — APPROVED** [review]

**Review by samajammin — APPROVED** [review]
👍

**Pull request merged**